docs/pages/origin_install.mdx

```diff
@@ -0,0 +1,74 @@
+import ExportedImage from "next-image-export-optimizer";
+
+# Install Pelican Origin
+
+The [Pelican](http://pelicanplatform.org/) origin exposes a storage backend like POSIX or S3 to other members of the federation. This will give you the steps to install XRootD as well as how to install Pelican and start up an origin.
+
+## Before Starting
+
+
+Before starting the installation process, consider the following points:
+
+-   **User IDs:** If it does not exist already, the installation will create the Linux user ID `xrootd`
+-   **Service certificate:** The XRootD service uses a host certificate and key pair at
+    `/etc/grid-security/xrd/xrdcert.pem` and `/etc/grid-security/xrd/xrdkey.pem` that must be owned by the `xrootd` user
+-   **Networking:** The XRootD service uses port 1094 by default
+
+As with all OSG software installations, there are some one-time (per host) steps to prepare in advance:
+
+-   Ensure the host has [a supported operating system](https://osg-htc.org/docs/release/supported_platforms/)
+-   Obtain root access to the host
+-   Prepare [the required Yum repositories](https://osg-htc.org/docs/common/yum/)
+-   Install [CA certificates](https://osg-htc.org/docs/common/ca/)
+-   Install XRootD (Instructions to follow)
+
+## Installing XrootD
+
+
+!!! warning "Requirements for XRootD-Multiuser with VOMS FQANs"
+    Using XRootD-Multiuser with a VOMS FQAN requires mapping the FQAN to a username, which requires a `voms-mapfile`.
+    Support is available in `xrootd-voms 5.4.2-1.1`, in the OSG 3.6 repos, though it is expected in XRootD 5.5.0.
+    If you want to use multiuser, ensure you are getting `xrootd-voms` from the OSG repos.
+
+To install an XRootD Standalone server, run the following command:
+
+```console
+root@xrootd-standalone # yum install osg-xrootd-standalone
+```
+
+## Installing Pelican
+
+
+Grab the appropriate binary for your system from the [pelican repository](https://github.com/PelicanPlatform/pelican/releases). For more details, see [the client installation instructions](/install.mdx)
+
+## Serving an Origin
+
+
+To launch a pelican origin, run:
+
+```./pelican origin serve -f <federation> -v <local_directory>:<namespace_prefix>```
+
+Where `<federation>` is the address of the federation the origin will be a part of, `<local directory>` is the directory containing objects to be exported to the federation, and `<namespace prefix>` is the namespace at which files from `<local directory>` will be made available in the federation.
+
+The first time the origin is started, you will see something that looks like the following:
+
+
+<ExportedImage width={1000} height={1000} src={"/pelican/origin_start.png"} alt={"Image of an origin startup terminal prompt with the initializaiton code"} />
```

Comment:
The image and other console examples all show Pelican being run as root - is this something we require, or is it just an artifact from our developing in a rootly container?